include/boost/geometry/algorithms/detail/envelope/implementation.hpp

```diff
@@ -43,27 +46,31 @@ namespace detail { namespace envelope
 {
 
 
-struct exterior_ring_expand_policy
-{
-    template <typename Box, typename Geometry>
-    static inline void apply(Box& mbr, Geometry const& geometry)
-    {
-        Box ring_mbr;
-        envelope_range<>::apply(exterior_ring(geometry), ring_mbr);
-        geometry::expand(mbr, ring_mbr);
-    }
-};
-
-
+template <std::size_t Dimension, std::size_t DimensionCount>
```

Comment:
In distance we do dispatching with respect to geometry types and strategies at the same time. Not distance (not yet at least as we still work on 2D).